pkgs/shells/zsh/zsh-z/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-z";
+  version = "2021-02-15";
```

Comment:
Should maybe be renamed to `unstable-2021-02-15`, as everything without a release, or especially before one can be considered that

Reply:
Yeah I was thinking this too, will rename it